R/python.R

```diff
@@ -1236,6 +1239,10 @@ py_callable_as_function <- function(callable, convert) {
       result
     }
   }
+  sig <- py_get_formals(callable, convert)
```

Comment:
Maybe wrap in `tryCatch()` and ignore errors? (since otherwise we just use `...` and things work as expected, just without explicit formals)

Reply:
Sure! Can I log errors somewhere in case we want to debug some signature that doesn’t convert even when it should?

Reply:
OK, I’m storing them in an attribute now.